modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,211 @@
 :toclevel: 2
 :openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
 :openid-connect-url: https://openid.net/connect/
+:openid-app-marketplace-url: https://marketplace.owncloud.com/apps/openidconnect
+:openid-app-github-url: https://github.com/owncloud/openidconnect
+:examplesdir: ../../../../examples/
 
 == Introduction
 
 "OpenID Connect 1.0 is a simple identity layer on top of the OAuth 2.0 protocol. It allows Clients to verify the identity of the End-User based on the authentication performed by an Authorization Server, as well as to obtain basic profile information about the End-User in an interoperable and REST-like manner.
 OpenID Connect allows clients of all types, including Web-based, mobile, and JavaScript clients, to request and receive information about authenticated sessions and end-users. The specification suite is extensible, allowing participants to use optional features such as encryption of identity data, discovery of OpenID Providers, and session management, when it makes sense for them."
 -- {openid-connect-url}, "What is OpenID Connect?"
 
+//ownCloud-specific OIDC description and benefits
+== OIDC Benefits
+
+== OIDC App Marketplace Link
+
+// Not currently available
+* https://marketplace.owncloud.com/apps/openidconnect
+
 == Prerequisites
 
-A distributed Memcache setup is required to operate this app - such as Redis or Memcached properly.
-Please follow xref:configuration/server/caching_configuration.adoc[the documentation] on how to set up caching.
+The OpenID Connect app has several mandatory requirements, these are:
+
+* A minimum PHP version of 7.0, with the CURL and JSON extensions.
```

Comment:
With 10.4, php7.0 was dropped, see changelog / releasenotes core #36290.
On docs, we recommend using 7.3 (Ubuntu 18.04 can default use only 7.2 and with ppa 7.3, see manual installation)
This is confusing to readers...

Reply:
Well spotted.